scripts/exrrfs_run_maxrh.ksh

```diff
@@ -0,0 +1,286 @@
+nn=0
+while [[ $nn -lt $num ]] ; do
+    let nnp1="nn+1"
+    dname=${gridsarray[$nn]}
+    echo "     gridnames($nnp1)=${dname}," >> gridsinfo_input
+    Tur=yes
+    run=rtma3d
+	Tur=yes
+	CYCLE=${PDY}1800
+        CYCLE_STOP=${PDY}0600
+	CYCLEm1=${PDY}1700
+
+    hr=1 #hour number to go in output (from wgrib2) file name
+    one=1
+    while [[ $CYCLE -ge $CYCLE_STOP ]] ; do
+	
+	YYYY=`echo $CYCLE | cut -c 1-4`
+	YYYYMM=`echo $CYCLE | cut -c 1-6`
+	YYYYMMDD=`echo $CYCLE | cut -c 1-8`
+	HH=`echo $CYCLE | cut -c 9-10`
+	YYYYMMDDm1=`echo $CYCLEm1 | cut -c 1-8`
+        HHm1=`echo $CYCLEm1 | cut -c 9-10`
+
+	
+	#find proper name of ges and analysis files to run wgrib2 on based on run and domain name
+       #find proper name of ges and analysis files to run wgrib2 on based on run and domain name
+        if [[ ${run} = "rtma3d" ]] ; then
+            if [[ $dname == "hrrr" ]] ; then
+                opsfile=${COMOUT_BASEDIR}/RTMA_NA.${YYYYMMDD}/${HH}/rtma.t${HH}z.prslev.f000.hrrr.grib2
+                gesfile=${RRFS_GRIBDIR}/rrfs.${YYYYMMDDm1}/${HHm1}/rrfs.t${HHm1}z.prslev.f001.conus_3km.grib2
+                if [[ ! -e $opsfile ]] ; then
+                  if [[ ! -d ${COMOUT_BASEDIR}/RTMA_NA.${YYYYMMDD}/${HH} ]]; then
+                    mkdir -p ${COMOUT_BASEDIR}/RTMA_NA.${YYYYMMDD}/${HH}
+                  fi
+	          opsfile=$gesfile
+	        fi
+	    fi
+        fi
+	#now use wgrib2 to pull ges/analysis from the file
+	#use run in binary filename, regardless of grid
+	if [[ -s $gesfile ]] ; then
+            if [ $hr -le 9 ]; then
+	    tmpops=${run}_temp_anl_hour_0${hr}.bin
+	    tmpges=${run}_temp_ges_hour_0${hr}.bin
+	    dptops=${run}_dwpt_anl_hour_0${hr}.bin
+	    dptges=${run}_dwpt_ges_hour_0${hr}.bin
+	    else
+	    tmpops=${run}_temp_anl_hour_${hr}.bin
+            tmpges=${run}_temp_ges_hour_${hr}.bin
+            dptops=${run}_dwpt_anl_hour_${hr}.bin
+            dptges=${run}_dwpt_ges_hour_${hr}.bin
+	    fi
+	    wgrib2 $opsfile -match ":TMP:2 m above ground:" -ieee $tmpops
+	    wgrib2 $gesfile -match ":TMP:2 m above ground:" -ieee $tmpges
+	    wgrib2 $opsfile -match ":DPT:2 m above ground:" -ieee $dptops
+	    wgrib2 $gesfile -match ":DPT:2 m above ground:" -ieee $dptges
+	else
+	    echo "WARNING: ${run} file unavailable for ${CYCLE}!";
+	fi	
+	CYCLE=`$MDATE -60 $CYCLE`
+        hr=$(( $hr + $one ))
+        if [ ${HH} -eq 06 ] ; then
+            if [[ ${run} == "rtma3d" ]] ; then
+                cp $gesfile gesfileus.grb2
+            fi
+        fi
+
+done #$CYCLE -le $CYCLE_STOP (number of cycles to run)
+    
+    let nn="$nn+1"
+done #$nn -lt $num (number of grids)
+
+
+echo "/" >> gridsinfo_input
+
+#make blend?  1=bgs only 2=anls only, 3=both
+export bnum=2
+cat <<EOF > blend_input
+&blendinput
+bnum=${bnum}
+/
+EOF
+
+#. prep_step
+
+ln -sf ${run}.${PDYm1}.maxrh_anl.dat fort.61
+ln -sf ${run}.${PDYm1}.maxrh_bg.dat fort.62
+
+export pgm=rtma_maxrh.exe
+#startmsg
+${EXECdir}/rrfs_maxrh.exe > stdout 2>&1
```

Comment:
To meet the NCO standards, this line should be `${EXECdir}/$pgm >>$pgmout 2>errfile` Please refer to the other scripts.